Validation/HGCalValidation/test/HGCalWaferHitCheck.cc

```diff
@@ -75,6 +77,7 @@ class HGCalWaferHitCheck : public edm::stream::EDAnalyzer<edm::GlobalCache<HGCal
   edm::EDGetTokenT<edm::PCaloHitContainer> tok_hit_;
   std::map<int, int> badTypes_, occupancy_;
   std::vector<int> goodChannels_;
+  std::mutex mtx_;
```

Comment:
i think this needs to be `static` so it locks across all instances of the analyzer (which all access the same globalCache object)